crypto/low_power.h

```diff
@@ -0,0 +1,6 @@
+#ifndef XMR_STAK_CPU_LOW_POWER_H
+#define XMR_STAK_CPU_LOW_POWER_H
```

Comment:
This should exposed to cmake and than optional guarded if someone is not using cmake and missed to set the define.

```
#ifndef XMR_STAK_LOW_POWER_HASHE_COUNT
#   define LOW_POWER_HASHES 2
#else
#   define LOW_POWER_HASHES XMR_STAK_LOW_POWER_HASHE_COUNT
#endif
```